networkit/test/test_distance.py

```diff
@@ -188,11 +237,48 @@ def testMultiTargetSTSP(self):
 					algo = nk.distance.MultiTargetDijkstra(g, source, targets)
 				else:
 					algo = nk.distance.MultiTargetBFS(g, source, targets)
-				algo.setTargets(targets)
+				algo.setSource(source)
+				algo.setTarget(targets[nTargets-1])
+				algo.run()
+				self.assertLessEqual(len(algo.getPredecessors()),g.numberOfNodes())
+
+	def testSSSP(self):
+		for g in self.genERGraphs():
+			source = nk.graphtools.randomNode(g)
+			for nTargets in [1, 10, 50]:
```

Comment:
Split in two tests: SSSP with single target, SSSP with multiple targets. For the latter, you could pick 3 fixed targets. We don't need such extensive tests in Python.